### PR TITLE
[New] Added test cases for pages export list

### PR DESCRIPTION
**Pull request by Srinivas9933**
Added test cases for pages export list component.

- container.spec.js

### CODE REVIEW

File: app/javascript/components/pages/export-list/container.spec.js [hunk 1]
```diff
@@ -96,39 +91,37 @@ describe("<ExportList />", () => {
...
   it("should render <PageHeading>", () => {
-    expect(component.find(PageHeading)).to.have.lengthOf(1);
+    mountedComponent(<ExportList />, {}, initialState);
+    expect(screen.getByTestId("page-heading")).toBeInTheDocument();
   });
 
   it("should render <PageContent>", () => {
-    expect(component.find(PageContent)).to.have.lengthOf(1);
+    mountedComponent(<ExportList />, {}, initialState);
+    expect(screen.getByText(/messages.record_list.rows_per_page/i)).toBeInTheDocument();
```
Comment (dhernandez-quoin) [inline app/javascript/components/pages/export-list/container.spec.js:111]: The `PageContent` component is not related to pagination. I think you will need a test-id.

File: app/javascript/components/pages/export-list/container.spec.js [hunk 2]
```diff
@@ -96,39 +91,37 @@ describe("<ExportList />", () => {
...
     const stateOffline = initialState.setIn(["application", "online"], false);
 
-    beforeEach(() => {
-      ({ component } = setupMountedComponent(ExportList, {}, stateOffline));
-    });
-
     it("should render DisabledOffline components for each row", () => {
-      expect(component.find(DisableOffline)).to.have.lengthOf(9);
+      mountedComponent(<ExportList />, stateOffline);
+      expect(screen.getAllByTestId("offline")).toHaveLength(3);
```
Comment (dhernandez-quoin) [inline app/javascript/components/pages/export-list/container.spec.js:124]: This no longer matches the original test.

File: app/javascript/components/pages/export-list/container.jsx [hunk 3]
```diff
@@ -55,6 +55,7 @@ const ExportList = () => {
                       {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events */}
                       <div
                         className={css.link}
+                        data-testid="offline"
```
Comment (dhernandez-quoin) [inline app/javascript/components/pages/export-list/container.jsx:58]: Can this be moved to the `DisableOffline` component?

**Review by dhernandez-quoin — CHANGES_REQUESTED** [review]

File: app/javascript/components/pages/export-list/container.spec.js (hunk 1 above)
Comment (Srinivas9933) [reply app/javascript/components/pages/export-list/container.spec.js]: Done

File: app/javascript/components/pages/export-list/container.jsx (hunk 3 above)
Comment (Srinivas9933) [reply app/javascript/components/pages/export-list/container.jsx]: Done

File: app/javascript/components/pages/export-list/container.spec.js (hunk 1 above)
Comment (dhernandez-quoin) [reply app/javascript/components/pages/export-list/container.spec.js]: Why was this test removed?

Comment (Srinivas9933) [reply app/javascript/components/pages/export-list/container.spec.js]: Done

File: app/javascript/components/pages/export-list/container.spec.js [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+  });
+
+  it("should render <PageContent>", () => {
+    mountedComponent(<ExportList />, {}, initialState);
+    expect(screen.getByTestId("page-content")).toBeInTheDocument();
+  });
+
+  it("should render <IndexTable>", () => {
+    mountedComponent(<ExportList />, {}, initialState);
+    expect(screen.getByText(/messages.record_list.rows_per_page/i)).toBeInTheDocument();
```
Comment (jtoliver-quoin) [inline app/javascript/components/pages/export-list/container.spec.js:117]: Need something else to key off of like role table or something

**Pull request merged**